### ISSUE TITLE
Suggestion: Inertia UI

### ISSUE THREAD

**Issue by Godeps2891** [THEREALWWEFAN231/Inertia#749]
**Minecraft Version(s)** all

**What is the suggestion** An option to add the transparent inertia main menu style buttons to every screen

**Why would this feature be useful** **_a e s t h e t i c s_**
since resource packs don't allow transparency for those elements

mockup:
![image](https://user-images.githubusercontent.com/54289108/93796754-9158b000-fc3b-11ea-883e-2fb91505859e.png)

**Comment by THEREALWWEFAN231** [comment]
I mean the buttons are pretty hawt.

**Comment by Paralusion** [comment]
agreed, would be even cooler if the color and transparency could be changed within the color manager ;)
good suggestion
